src/index.js

```diff
@@ -1,2 +1,41 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+  return element;
```

Comment:
```suggestion
  Object.entries(props || {}).forEach(([key, value]) => {
    element[key.toLowerCase()] = value;
  });
  
  children.flat().forEach((child) => {
    if (child instanceof Node) {
      element.appendChild(child);
      return;
    }
    element.appendChild(document.createTextNode(child));
  });
  
  return element;
```

저는 block 이 종료되는 지점 에서 한칸씩 띄어서 적는게 읽기 편하신데 수호님은 어떠신가요?

Reply:
저도 그렇게 생각합니다. 이 부분도 강의 코드를 그대로 써서 신경을 쓰지 못했습니다 ㅜㅜ
다음부터는 적용하도록 하겠습니다!

Reply:
저는 PR 을 날리기 전에 file change 를 한번 보고 스스로 리뷰를 하는것이 실수를 찾는데 도움이 많이 되더라구요!! 수호님도 한번 PR 을 날리기 전에 셀프 리뷰를 해보세요! :)

Reply:
네 다음부터 PR 날리기 전에 스스로 리뷰를 꼭 해보겠습니다:)